website/package.json

```diff
@@ -4,23 +4,24 @@
   "version": "1.0.0",
   "author": "HashiCorp",
   "dependencies": {
-    "@hashicorp/mktg-global-styles": "2.1.0",
-    "@hashicorp/nextjs-scripts": "16.3.0",
-    "@hashicorp/react-button": "4.0.0",
-    "@hashicorp/react-docs-page": "12.0.0",
-    "@hashicorp/react-hashi-stack-menu": "1.2.1-canary.0",
-    "@hashicorp/react-head": "1.1.6",
-    "@hashicorp/react-inline-svg": "5.0.0",
-    "@hashicorp/react-markdown-page": "^0.1.0",
+    "@hashicorp/mktg-global-styles": "3.0.1",
+    "@hashicorp/mktg-logos": "1.0.2",
+    "@hashicorp/nextjs-scripts": "17.1.0",
+    "@hashicorp/react-button": "5.0.0",
+    "@hashicorp/react-docs-page": "13.0.3",
+    "@hashicorp/react-hashi-stack-menu": "2.0.0",
+    "@hashicorp/react-head": "2.0.0",
+    "@hashicorp/react-inline-svg": "6.0.0",
+    "@hashicorp/react-markdown-page": "1.0.0",
     "@hashicorp/react-product-downloader": "4.0.2",
```

Comment:
I think we need a bump here as well - seeing some low-contrast and mismatch issues on [the `Download` page](https://packer-msian6zlp-hashicorp.vercel.app/downloads).

Might even make sense to jump right to the [new `product-downloads-page` component](https://github.com/hashicorp/react-components/releases/tag/%40hashicorp%2Freact-product-downloads-page%401.0.0), though not sure if the upgrade level-of-effort there makes sense or not.... actually I might even take that back, definitely makes sense to just bump the branding version here.

<img width="1275" alt="product-downloader-packer-brand-issue" src="https://user-images.githubusercontent.com/4624598/113782935-f5872100-9700-11eb-9730-f85da92aa0da.png">

Reply:
Yeah I was under the impression @jescalan is in the middle of rolling out product-downloader updates, so I left that one out. Maybe it's seeming like we roll out these two updates together?

Reply:
☝️ Note on the above - I didn't realize we were 5 major versions behind, so per some Slack chat a CSS override here might be the best way forward to keep the scope of this PR sane.

We'll likely be following up with the `product-downloads-page` update pretty quickly, so the overrides feel like a pretty minor, short-lived, and worthwhile compromise.

Reply:
<img width="1221" alt="Screen Shot 2021-04-06 at 3 37 41 PM" src="https://user-images.githubusercontent.com/36613477/113786472-09755780-96ee-11eb-8664-66bdd6dbd437.png">
Added a few overrides

Reply:
Oh should that download button be the corporate blue actually?